src/routes/profile/index.tsx

```diff
@@ -66,19 +66,20 @@ router.get('/avatargetlinktutorial', (req, res) => {
 // Show the mod approving rejecting page
 router.get('/mod/customavatar', isModMiddleware, async (req, res) => {
   const customAvatarRequests = await avatarRequest.find({ processed: false });
-  const avatarLibraryReact = renderToString(<AvatarLibrary />);
+  const avatarLookupReact = renderToString(<AvatarLookup />);
 
   res.render('mod/customavatar', {
     customAvatarRequests,
     MAX_FILESIZE_STR,
     VALID_DIMENSIONS_STR,
-    avatarLibraryReact,
+    avatarLookupReact,
   });
 });
 
-export interface AllUserAvatars extends AllApprovedAvatars {
+export interface AllUserAvatars {
   currentResLink: string | null;
   currentSpyLink: string | null;
+  allApprovedAvatars: AllApprovedAvatars;
 }
 
 // Get all the approved avatars for a user. Only available to mods
```

Comment:
Let's save this for the future, but let's move this to a public area in the future so that the gallery can view all of a user's approved avatars for all of time.